<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleQtClass</class>
 <widget class="QMainWindow" name="SimpleQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>661</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_right">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>661</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_center">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>661</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="simpleqt.qrc"/>
 </resources>
 <connections/>
</ui>
